test/main/tetris_piece_test.cpp

```diff
@@ -5,12 +5,44 @@
 #include "render_component.h"
 #include "SDL_render.h"
 #include "render_component_mock.h"
+#include "tetris_piece.h"
+#include "trompeloeil.hpp"
 #include <memory>
+#include <vector>
 
 using std::unique_ptr;
 using std::make_unique;
 using std::move;
+using std::vector;
 using trompeloeil::_;
+using trompeloeil::expectation;
+
+vector<unique_ptr<expectation>> set_rotate_expectations( unique_ptr<TetrisPiece>& piece )
```

Comment:
It may be worth it to add convenience methods like these to a "test header" for an individual testing file. This works well enought when there are only a couple such methods.